hooks/open-telemetry/pkg/traces.go

```diff
@@ -3,20 +3,25 @@ package otel
 import (
 	"context"
 	"fmt"
+	"strings"
 
 	"github.com/open-feature/go-sdk/openfeature"
 	"go.opentelemetry.io/otel/attribute"
-	"go.opentelemetry.io/otel/codes"
 	semconv "go.opentelemetry.io/otel/semconv/v1.34.0"
```

Comment:
I know it's not part of your PR, but v1.37.0 is the latest.
https://pkg.go.dev/go.opentelemetry.io/otel/semconv